sql.txt

```diff
@@ -3,9 +3,16 @@ SELECT with JOIN practice:
 Join the OrderDetails and Products tables, to show a report where the columns are OrderID, ProductName, and Quantity.
 Paste the SQL statement you used below.
 
+SELECT OrderID, ProductName, Quantity FROM OrderDetails
+	INNER JOIN Products ON OrderDetails.ProductID = Products.ProductID;
```

Comment:
This is correct. One thing you can do is set aliases for the tables. for example you could have FROM OrderDetails AS o. Then you could say o.ProductID instead of OrderDetails.ProductID. You can do that for each table you reference, which saves a lot of time when referencing a lot of tables.